flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/stream/table/validation/AggregateValidationTest.scala

```diff
@@ -47,4 +47,54 @@ class AggregateValidationTest extends TableTestBase {
       // must fail. 'c is not a grouping key or aggregation
       .select('c)
   }
+
+  @Test(expected = classOf[ValidationException])
+  def testTableFunctionInSelection(): Unit = {
```

Comment:
This test failed with the exception:
org.apache.flink.table.api.ValidationException: Given parameters of function 'func' do not match any signature. 
Actual: (java.lang.Long) 
Expected: (java.lang.String)